bodo/pandas/_executor.h

```diff
@@ -0,0 +1,58 @@
+// Executor backend for running DataFrame library plans in DuckDB format
+
+#pragma once
+
+#include <Python.h>
+#include "duckdb/planner/operator/logical_get.hpp"
+
+/**
+ * @brief Physical operators to be used in the execution pipelines (NOTE: they
+ * are Bodo classes and not using DuckDB).
+ *
+ */
+class PhysicalOperator {
+   public:
+    virtual std::pair<int64_t, PyObject*> execute() = 0;
```

Comment:
Why use a pair here rather than make a class?  A class could be better self-documenting.

Reply:
It's passed to Cython so a bit easier. This is just a placeholder and will probably change soon anyways.